<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvestmentWindow</class>
 <widget class="QMainWindow" name="InvestmentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>520</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Investment Account Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget_test">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>0</y>
      <width>231</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Balance :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="recipientUsername"/>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="logout">
       <property name="text">
        <string>Log out</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="balanceLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Recipient's username :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="3">
      <widget class="QPushButton" name="transferButton">
       <property name="text">
        <string>Transfer</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Amount :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Username :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="amount"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select stock</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AAPL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MSFT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TSLA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GOOGL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AMZN</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="share_price">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>current share value :</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>231</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">#label_5{
font: bold;
}</string>
       </property>
       <property name="text">
        <string>BUY AND SELL STOCKS :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>201</width>
           <height>391</height>
          </rect>
         </property>
         <property name="title">
          <string>CHOOSE AND BUY!</string>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Select Stock :</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_buy">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>161</width>
            <height>32</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Select stock</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AAPL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MSFT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TSLA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GOOGL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AMZN</string>
           </property>
          </item>
         </widget>
         <widget class="QLineEdit" name="share_amount_buy">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>160</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>181</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Select the number of share :</string>
          </property>
         </widget>
         <widget class="QPushButton" name="buyButton">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>220</y>
            <width>100</width>
            <height>32</height>
           </rect>
          </property>
          <property name="text">
           <string>BUY</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>201</width>
           <height>401</height>
          </rect>
         </property>
         <property name="title">
          <string>SELECT AND SELL!</string>
         </property>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Select Stock :</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_sell">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>161</width>
            <height>32</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Select Stock</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AAPL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MSFT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TSLA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GOOGL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AMZN</string>
           </property>
          </item>
         </widget>
         <widget class="QLineEdit" name="share_amount_sell">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>160</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>181</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Select the number of share :</string>
          </property>
         </widget>
         <widget class="QPushButton" name="sellButton">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>220</y>
            <width>100</width>
            <height>32</height>
           </rect>
          </property>
          <property name="text">
           <string>SELL</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="share_value">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>161</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Owned Stocks :</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>60</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>AAPL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>MSFT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>TSLA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>AMZN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>GOOGL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>60</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="apple_amount">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="microsoft_amount">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tesla_amount">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="amazon_amount">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="google_amount">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>420</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1282</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>logout</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>transferButton</tabstop>
  <tabstop>amount</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>recipientUsername</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
